JS/Lesson13/sinan/script.js

```diff
@@ -2,143 +2,177 @@
 // The reason the output is one hour behind what you might expect (i.e. January 1, 2000 at 00:00:00) is because of the timezone offset between your local timezone and UTC. The output indicates that your local timezone is one hour ahead of UTC. This is likely because your local timezone observes daylight saving time, which can cause the timezone offset to change depending on the time of year.
 // To get the date and time in your local timezone, you can use methods such as getDate(), getMonth(), and getFullYear() to extract the individual components of the date and time, and then construct a new Date object using those components. 
 
+
+const now = new Date;
+console.log('Exercises9 :',now);
+
+
+
 console.log('------------------------');
+
 // ---
 // 10. Declare a variable number6 and initialize it to 6. Declare a variable number7 and initialize it to `++number6` (number6 incremented using prexif ++ operator). Log both values to the console.
-let number6 = 6;
 // First, number6 got incremented, then number7 got value of number6 (it was 7);
-const number7 = ++number6; // 7
-console.log('Exercise 10:', number6, number7);
-const number99 = ++number6 + 1; // 8
 
-console.log('Exercise 10:', number6, number7, number99);
+let number6 = 6;
+const number7 = ++ number6;
+const number99 = ++number6 + 1;
+console.log('Exercises10.1 :', number6,number7);
+console.log('Exercises10.2 :', number6,number7,number99);
+
 
 console.log('------------------------');
+
 // ---
 // 11. Declare a variable number8 and initialize it to 8. Declare a variable number9 and initialize it to `number8++` (number8 incremented using postfix ++ operator). Log both values to the console.
-let number8 = 6;
+
 // First, number9 got value of number8 (it was 6), then number8 got incremented;
-let number9 = number8 + 1 ; //6
-console.log('Exercise 11:', number8, number9);
+
+let number8 = 6 ;
+let  number9 = number8+1;
+console.log('Exercises11 :', number8,number9);
+
+
 
 console.log('------------------------');
 
 // ---
 // 12. Declare a variable number10 and initialize it to -3. Use the postfix ++ operator to increment the value of num by 1, then multiply the result by 2 and log it to the console.
+
 let number10 = -3;
-console.log(number10++ * 2);
-console.log('Exercise 12:',number10);
+let newNumber = number10++;
+let newNumber2 = newNumber*2;
+console.log('Exercises12.1 :',number10++ * 2);
+console.log('Exercises12.2 :',newNumber * 2);
+console.log('Exercises12.3 :',newNumber2);
 
 console.log('------------------------');
 // ---
 // 13. Declare two variables a and b and initialize them to 2 and 3, respectively. Use the prefix ++ operator to increment the value of a by 1, then add b to the result and log it to the console.
+
 let a = 2;
-const b = 3;
-console.log('Exercise 13:',++a + b);
+console.log('Exercises13.1 :', a);
+let b = 3;
+b = ++a;
```

Comment:
Requirement is a bit different here: "Use the prefix ++ operator to increment the value of a by 1, then add b to the result and log it to the console."```suggestion
++a;
const result = a + b;
```